Create security/scams learn page (#3872)
ethereum/ethereum-org-website: src/content/security/index.md

---
title: Ethereum security and scams
description: Staying safe on Ethereum
lang: en
sidebar: true
---

## Staying Safe on Ethereum

With interest in cryptocurrencies growing exponentially, learning best practices when using cryptocurrency is essential. Crypto can be fun and exciting, but there are also serious risks. If you put in this small amount of upfront work, you can mitigate these risks.

<Divider />

## Web Security 101

### Use strong passwords

Over 80% of account hacks are a result of weak or stolen passwords. A long combination of characters, numbers and symbols is best to keep your accounts secure.

> A common mistake individuals make is using a long combination of common dictionary words. Passwords like this are insecure because they are prone to a simple hacking technique known as a [dictionary attack](<[https://en.wikipedia.org/wiki/Dictionary_attack](https://en.wikipedia.org/wiki/Dictionary_attack)>).

Author: Yeah I kinda missed the point I was making. Fixed to say this:

> A common mistake individuals make is using a combination of two to three common, related dictionary words. Passwords like this are insecure because they are prone to a simple hacking technique known as a [dictionary attack](<[https://en.wikipedia.org/wiki/Dictionary_attack](https://en.wikipedia.org/wiki/Dictionary_attack)>).

I think all of the information you suggested is probably suitable for an advanced section. Happy to iterate on this with you because you're definitely more knowledgeable.

```md
Example of a weak password: FluffyKittens!

Example of a strong password: ymv\*azu.EAC8eyp8umf
```

#### Good password practices:

- Make passwords **at least** 12 characters long
- Use a mixture of uppercase, lowercase, numbers and symbols
- Avoid common dictionary words

[More on creating strong passwords](https://terranovasecurity.com/how-to-create-a-strong-password-in-7-easy-steps/)

### Use unique passwords for everything

A strong password doesn't provide as much protection if the password is revealed in a data breach. The website [Have I Been Pwned](<notion://www.notion.so/efdn/%5B%3Chttps://haveibeenpwned.com/%3E%5D(%3Chttps://haveibeenpwned.com/%3E)>) allows you to check if your accounts were involved in any data breaches stored in their database. If they have, **you should change pwned passwords immediately**. Using unique passwords for every account lowers the risk of hackers getting access to all of your accounts when one of your passwords is compromised.

> ### Use a password manager

Author: Hmmm. If a user takes nothing else from this page other than downloading a password manager it's probably a win. I've added some emphasis to this section to push that.

![Screenshot 2021-09-14 at 11 25 15](https://user-images.githubusercontent.com/62268199/133241505-ce702ae7-571a-4a82-85c7-b2794acc673a.png)

At the same time, I think going through this should be a linear journey. (i.e. IRL people learn to use symbols and numbers before they make the jump to password managers). Unfortunately, a lot of people won't download a pw manager because of the overhead, I'm not sure moving this to position #1 helps that.

I'm kinda on the fence here - can we discuss this more at standup @wackerow? Would love some other opinions on this.

cc: @lukassim @pettinarip @corwintines @samajammin

> ### Use a password manager

Reviewer: I think you both have good points here. Will do some thinking about it.

> ### Use a password manager

Reviewer 2: @minimalsm Since you've mentioned the costs connected with password managers, would it also make sense to mention which ones offer free plans? I know LastPass and Bitwarden do, while KeePass seems to be completely free.

> ### Use a password manager

Author: > Since you've mentioned the costs connected with password managers, would it also make sense to mention which ones offer free plans? I know LastPass and Bitwarden do, while KeePass seems to be completely free.

@lukassim sorry I meant cognitive overhead — “how many logical connections or jumps your brain has to make in order to understand or contextualize the thing you’re looking at.”

But good idea we should work out a fair way to differentiate them based on price.

Remembering strong, unique passwords for every account you have isn't ideal. A password manager offers a secure, encrypted store for all your passwords that you can access through one strong master password. They also suggest strong passwords when signing up for a new service, so you don't have to create your own. Many password managers will also tell you if you have been involved in a data breach, allowing you to change the passwords before any malicious attacks.

![Example of using a password manager](./passwordManager.png)

#### Try a password manager:

- [LastPass](<[https://www.lastpass.com/](https://www.lastpass.com/)>)
- [1Password](<[https://1password.com/](https://1password.com/)>)

> - [KeePass](<[https://keepass.info/](https://keepass.info/)>)

Author: > and supports several security features including security keys and customizing how many times your password is hashed before being stored (making rainbow attacks and brute-force significantly more difficult).

Good idea to add this to an advanced section/page when we iterate on this.

### Use Two-Factor Authentication

Using **Two-Factor Authentication (2FA)** provides an additional *security factor* for your online accounts so that knowing your password alone is not enough to access an account. Most commonly, this is a randomised 6-digit code that you can access through an authenticator app.

<InfoBanner emoji=":lock:">
  <div>
    Note: Using SMS-based 2FA is susceptible to 
    <a href="https://www.vice.com/en/article/3kx4ej/sim-jacking-mobile-phone-fraud">
      SIM jacking
    </a>
     and is not secure. For the best security, use a service like{" "}
    <a href="https://mashable.com/article/how-to-set-up-google-authenticator">
      Google Authenticator
    </a>
     or <a href="https://authy.com/">Authy</a>.
  </div>
</InfoBanner>

[More on 2FA](<[https://authy.com/what-is-2fa/](https://authy.com/what-is-2fa/)>)

> 

Reviewer 3: https://youtu.be/m8jlnZuV1i4?t=3479 (Andreas video, timestamped to discussion on 2FA)

> 

Author: Added video embed and explained factors

---
title: Ethereum security and scams
description: Staying safe on Ethereum
lang: en
sidebar: true
---

## Staying Safe on Ethereum

> With interest in cryptocurrencies growing exponentially, learning best practices when using cryptocurrency is essential. Crypto can be fun and exciting, but there are also serious risks. If you put in this small amount of upfront work, you can mitigate these risks.

Reviewer 4: ```suggestion
With interest in cryptocurrencies growing, learning best practices when using cryptocurrency is essential. Crypto can be fun and exciting, but there are also serious risks. If you put in this small amount of upfront work, you can mitigate these risks.
```
Not sure we should say exponential growth - at some point this might not be the case.

Author: > at some point this might not be the case.

Hopefully it does but agreed 🤪

  <div>
    Using a password manager takes care of creating strong, unique passwords and remembering them! We <strong>strongly</strong> recommend using one.
  </div>
</InfoBanner>

Remembering strong, unique passwords for every account you have isn't ideal. A password manager offers a secure, encrypted store for all your passwords that you can access through one strong master password. They also suggest strong passwords when signing up for a new service, so you don't have to create your own. Many password managers will also tell you if you have been involved in a data breach, allowing you to change the passwords before any malicious attacks.

![Example of using a password manager](./passwordManager.png)

#### Try a password manager:

- [Bitwarden](https://bitwarden.com/)
- [KeePass](https://keepass.info/)
- [LastPass](https://www.lastpass.com/)
- [1Password](https://1password.com/)

### Use Two-Factor Authentication

To prove you are you there are different unique proofs that can be used for authentication. These are known as factors and the three main factors are:

- something you know (your password)
- something you are (such as a fingerprint or iris scanner)

> - and something you own (a security key)

Reviewer 2: All the other lists are capitalized, this seems to be the only one starting with lowercase letters.

Using **Two-Factor Authentication (2FA)** provides an additional *security factor* for your online accounts so that knowing your password alone (something you know) is not enough to access an account. Most commonly, the second factor is a randomised 6-digit code (something you own) that you can access through an authenticator app such as Google Authenticator or Authy. These work as a "something you own" factor because the seed is stored on your device.

<InfoBanner emoji=":lock:">
  <div>
    Note: Using SMS-based 2FA is susceptible to 
    <a href="https://www.vice.com/en/article/3kx4ej/sim-jacking-mobile-phone-fraud">
      SIM jacking
    </a>
     and is not secure. For the best security, use a service like{" "}
    <a href="https://mashable.com/article/how-to-set-up-google-authenticator">

>       Google Authenticator

Reviewer 2: While authenticator apps are the most popular and many people likely consider a security key too expensive, it would probably still make sense to mention them and note that this is the safest 2FA option.

    </a>
     or <a href="https://authy.com/">Authy</a>.
  </div>
</InfoBanner>

Watch more on the 2FA:

<iframe width="100%" height="315px" src="https://www.youtube.com/embed/m8jlnZuV1i4?start=3479&end=3875" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

### Uninstall browser extentions

Browser extensions like Chrome extensions or Add-ons for Firefox can augment useful browser functionality and improve user experience, but they come with risks. By default, most browser extensions ask for access to 'read and change site data', allowing them to do almost anything with your data. Chrome extensions are always automatically updated, so a previously safe extension may update later to include malicious code. Most browser extensions are not trying to steal your data, but you should be aware that they can.

#### Stay safe by:

- Only install browser extensions from trusted sources
- Removing unused browser extensions
- Install Chrome extensions locally to stop auto-updating (Advanced)

> [More on the risks of browser extentions](<[https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/](https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/)>)

Reviewer 4: ```suggestion
[More on the risks of browser extentions](https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/)
```
Broken link

<Divider />

## Crypto Security 101

### Upgrade Your Knowledge

> One of the biggest reasons people get scammed in crypto generally is a lack of understanding. If you don't understand that no one owns the Ethereum network, then it's easy to fall prey to the kind customer service agent who promises to return your lost ETH in exchange for your private keys. Educating yourself on how Ethereum works is a worthwhile investment.

Reviewer: ```suggestion
One of the biggest reasons people get scammed in crypto generally is a lack of understanding. For example, if you don't understand that the Ethereum network is decentralized and owned by no one, then it's easy to fall prey to someone pretending to be a customer service agent that promises to return your lost ETH in exchange for your private keys. Educating yourself on how Ethereum works is a worthwhile investment.
```

<DocLink to="/what-is-ethereum/" title="What is Ethereum?" />

<DocLink to="/eth/" title="What is ether?" />

> 

Reviewer: ```suggestion
<Divider />
```
Add this in here to separate sections like the rest of the page does

> ## Wallet Security

Reviewer 4: ```suggestion

## Wallet Security
```

### Don't give out your Private keys

> **Never, for anyone reason, hand out your private keys!**

Reviewer 4: ```suggestion
**Never, for any reason, share your private keys!**
```

Reviewer 4: Small type - also "hand out" may get lost in translation.

> The private key to your wallet acts as your digital signature. It is the only thing stopping someone who knows your wallet address from draining your account of all of its funds!

Reviewer: ```suggestion
The private key to your wallet acts as your digital signature. It is the only thing stopping someone who knows your wallet address from draining your account of all of its assets!
```
Changing to assets with the rise of NFT's

> The private key to your wallet acts as your digital signature. It is the only thing stopping someone who knows your wallet address from draining your account of all of its assets!

Reviewer 4: Does it make sense to also compare it to a password? As in it's the password to your Ethereum account. Or to your funds yourself - i.e. whoever has your private key has control of your funds!

Author: I did feel the wording of this was clunky. Comparing it to a password is probably a good heuristic to use.

> ### Use a Hardware Wallet

Reviewer 4: Worth adding a link to the wallets page to learn what wallets are - similar to how you linked to What is Ethereum & What is ETH pages above.

A hardware wallet provides offline storage for private keys. They are considered the most secure wallet option for storing your private keys.

Keeping private keys offline massively reduces the risk of being hacked, even if a hacker gets control of your computer.

#### Try a hardware wallet:

- [Ledger](<[https://www.ledger.com/](https://www.ledger.com/)>)
- [Trezor](<[https://trezor.io/](https://trezor.io/)>)

> ### Double check transactions

Reviewer 4: Double check before sending transactions?

> Accidently sending crypto to the wrong wallet address is a common mistake. Unless you know the address owner, there will be no way for you to retrieve your funds.

Reviewer 4: Perhaps worth mentioning that transaction on Ethereum (such as sending ETH) are irreversible?

Always make sure the address you are sending to exactly matches the desired recipient's address before sending a transaction.

> 

Reviewer: ```suggestion
It is also recommended when interacting with a smart contract to read the transaction message before signing.
```
Adding this as an idea due to the airdrop scam I heard about: https://youtube.com/watch?v=LLL_nQp1lGk

> Always make sure the address you are sending to exactly matches the desired recipient's address before sending a transaction.

Reviewer 4: Might be a worthy time to recommend ENS (or wallets that provide that, like Argent) to avoid the fat finger address problem?

> ### Set smart contract spend limits

Reviewer 4: Similarly, using a wallet with spending limits can be handy.

When interacting with smart contracts, do not allow unlimited spend limits. An unlimited spend could enable the smart contract to drain your wallet. Instead, set spending limits to only the amount necessary for the transaction.

> ### Don't take screenshots of your seed phrases/private keys

Reviewer 4: This section might be better within the "don't share you private keys" section?

By screenshotting your seed phrases or private keys, you risk syncing them to the cloud and potentially making them accessible to hackers. Obtaining private keys from the cloud is a common attack vector for hackers.

<Divider />

## Common Scams

> Todo: add copy here

Reviewer: I think we should mention something about rugpulls here too given the prominence of influencers promoting scam coins. Ex: Kim Kardashian and Floyd Mayweather promoting Ethereum Max.

Reviewer 4: Good point. "Do your own research" is annoying but always useful advice. In this space you can't always rely on a "credible" person or organization to tell you what to do.

> ### Giveaway scam (Double your ETH)

Reviewer 4: I think we could mention in here: No One Is Giving You Free or Discounted ETH

One of the most common scams in cryptocurrency is the giveaway scam. The giveaway scam can take many forms, but the general premise is that if you send ETH to the provided wallet address, you will receive your ETH back but doubled. *For this reason, it is also known as the 2-for-1 scam.*

These scams usually stipulate a limited time of opportunity to claim the giveaway to encourage poor decision-making and create a sense of false urgency.

> #### Twitter Hack

Reviewer 4: Good example. I might broaden this to "social media hacks"? Another pretty common one is for a hacker to hijack a popular YouTube channel (e.g. with millions of subscribers) then use it to start pushing scams.

Reviewer 5: Ok stop it in it's tracks

A high-profile version of this occurred in July 2020, when the Twitter accounts of prominent celebrities and organisations got hacked. The hacker simultaneously posted a Bitcoin giveaway on the hacked accounts. Although the deceptive tweets were quickly noticed and deleted, the hackers still managed to get away with 11 bitcoin (or $500,000 as of September 2021)

![A scam on Twitter](./twitterScam.png)

> #### Vitalik Giveaway

Reviewer 4: Or any celebrity?

Author: Makes sense. 

I tried to pull some Algolia data for this but I couldn't search the range I wanted to check. My intuition here was that most people come to discord asking specifically about giveaways that Vitalik is doing and they might query search engines the same way. My hope was to capture that traffic. Could use your knowledge on how we could best do this from an SEO perspective.

Vitalik is used most often in this scam, but many other prominent people involved in crypto are also used (e.g. Elon Musk or Charles Hoskinson). Including a well-known person gives the scammers livestream a sense of legitimacy (this looks sketchy, but Vitalik is involved, so it must be ok!).

**Giveaways are always scams. If you send your funds to these accounts, you will lose them forever.**

![A scam on YouTube](./youtubeScam.png)

### Support Scams

Cryptocurrency is a relatively young and misunderstood technology. A common scam that takes advantage of this is the support scam, where scammers will impersonate support personnel for popular wallets, exchanges, or blockchains.

Much of the discussion about Ethereum happens on Discord. Support scammers will commonly find their target by searching for support questions in public discord channels and then sending the enquirer a private message offering support. By building trust, support scammers try to trick you into revealing your private keys or sending your funds to their wallets.

> ![A support scam on Discord](./supportScam.png)

Reviewer 5: Ok how. I don't know what I'm doing.

As a general rule, staff will never communicate with you through private, unofficial channels. Some simple things to keep in mind when dealing with support:

- Never share your private keys, seed phrases or passwords
- Never allow anyone remote access into your computer

> - Never communicate in unofficial private channels

Reviewer 4: Is "unofficial" / "official" a word we want to use?

Author: Probably not the best language. What I'm trying to say here is that a support chat on an official website is ok, a discord DM is not.

Reviewer: Would undesignated be better?

<InfoBanner emoji=":lock:">
  <div>
    Beware: although support-style scams commonly happen on Discord, they can also be prevalent on any chat applications where crypto discussion happens, including email.
  </div>
</InfoBanner>

> ### Phishing Scams

Reviewer 4: Good resource to potentially mention here: https://support.mycrypto.com/staying-safe/mycrypto-protips-how-not-to-get-scammed-during-ico

Phishing scams are another increasingly common angle that scammers will use to attempt to steal your wallet's funds.

Some phishing emails ask users to click on links that will re-direct them to imitation websites, asking them to enter their seed phrase, reset their password or send ETH. Others may ask you to unknowingly install malware to infect your computer and give scammers access to your computer's files.

If you receive an email from an unknown sender, remember:

> - Never open a link or attachment from email addresses you don't recognise

Reviewer 4: ```suggestion
- Never open a link or attachment from email addresses you don't recognize
```

Reviewer 4: 😜

Author: ew

These fraudulent brokers find their targets by using fake accounts on YouTube to start seemingly natural conversations about the broker. These conversations are often highly upvoted to increase legitimacy, but the upvotes are all from bot accounts.

**Do not trust internet strangers to invest on your behalf. You will lose your crypto.**

![A trading broker scam on YouTube](./brokerScam.png)

<Divider />

## Further Reading

### Web Security

- [This is why you shouldn’t use texts for two-factor authentication](https://www.theverge.com/2017/9/18/16328172/sms-two-factor-authentication-hack-password-bitcoin)
- [Up to 3 million devices infected by malware-laced Chrome and Edge add-ons](https://arstechnica.com/information-technology/2020/12/up-to-3-million-devices-infected-by-malware-laced-chrome-and-edge-add-ons/)
- [How to Create a Strong Password — That You Won’t Forget](https://www.avg.com/en/signal/how-to-create-a-strong-password-that-you-wont-forget)

> ### Crypto Security

Reviewer 4: Another great resource: https://support.mycrypto.com/staying-safe/protecting-yourself-and-your-funds

Reviewer 4: Also: https://medium.com/mycrypto/mycryptos-security-guide-for-dummies-and-smart-people-too-ab178299c82e

Reviewer 4: A couple good suggestions from there that perhaps we should add:
- Don't use a clipboard manager
- Bookmark Your Crypto Sites / Do Not Trust Random URLs / Always Verify that the Site You Landed on Is Legit

Reviewer 4: Also https://support.mycrypto.com/staying-safe/common-scams

- [Crypto Security: Passwords and Authentication](https://www.youtube.com/watch?v=m8jlnZuV1i4)
- [4 Ways to Stay Safe in Crypto](https://www.coindesk.com/tech/2021/04/20/4-ways-to-stay-safe-in-crypto/)

> 

Reviewer 2: ```suggestion
- [MyCrypto’s Security Guide For Dummies And Smart People Too](https://medium.com/mycrypto/mycryptos-security-guide-for-dummies-and-smart-people-too-ab178299c82e)

```
Here's another in-depth crypto security guide we could add.

Reviewer 5: Let's do it

  </div>
</InfoBanner>

Watch more on the 2FA:

<iframe width="100%" height="315px" src="https://www.youtube.com/embed/m8jlnZuV1i4?start=3479&end=3875" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

### Uninstall browser extentions

Browser extensions like Chrome extensions or Add-ons for Firefox can augment useful browser functionality and improve user experience, but they come with risks. By default, most browser extensions ask for access to 'read and change site data', allowing them to do almost anything with your data. Chrome extensions are always automatically updated, so a previously safe extension may update later to include malicious code. Most browser extensions are not trying to steal your data, but you should be aware that they can.

#### Stay safe by:

- Only install browser extensions from trusted sources
- Removing unused browser extensions
- Install Chrome extensions locally to stop auto-updating (Advanced)

[More on the risks of browser extentions](https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/)

<Divider />

## Crypto Security 101

> ### Upgrade Your Knowledge

Author: ```suggestion
### Upgrade your knowledge
```

---

> title: Ethereum security and scams

Reviewer 4: ```suggestion
title: Ethereum security and scam prevention
```

Reviewer 4: I like the idea of focusing on "scam prevention" - LMK what you think

description: Staying safe on Ethereum
lang: en
sidebar: true
---

> ## Ethereum security and scams {#introduction}

Reviewer 4: ```suggestion
## Ethereum security and scam prevention {#introduction}
```

With interest in cryptocurrencies growing, learning best practices when using cryptocurrency is essential. Crypto can be fun and exciting, but there are also serious risks. If you put in this small amount of upfront work, you can mitigate these risks.

<Divider />

## Web security 101 {#web-security}

### Use strong passwords {#use-strong-passwords}

[Over 80% of account hacks are a result of weak or stolen passwords](https://cloudnine.com/ediscoverydaily/electronic-discovery/80-percent-hacking-related-breaches-related-password-issues-cybersecurity-trends/). A long combination of characters, numbers and symbols is best to keep your accounts secure.

A common mistake individuals make is using a combination of two to three common, related dictionary words. Passwords like this are insecure because they are prone to a simple hacking technique known as a [dictionary attack](https://en.wikipedia.org/wiki/Dictionary_attack).

```md
Example of a weak password: CuteFluffyKittens!

Example of a strong password: ymv\*azu.EAC8eyp8umf
```

> Another common mistake is using passwords that can be easily guessed or found out through [social engineering](<https://en.wikipedia.org/wiki/Social_engineering_(security)>). Including your mother's maiden name, the name's of your children or pets, or dates of birth in your password is not secure and will increase the risk of your password getting hacked.

Reviewer 6: I found a non-wiki [article ](https://www.webroot.com/us/en/resources/tips-articles/what-is-social-engineering) about social engineering. It does self advertise at the end but otherwise it looks good.

### Uninstall browser extentions {#uninstall-browser-extentions}

Browser extensions like Chrome extensions or Add-ons for Firefox can augment useful browser functionality and improve user experience, but they come with risks. By default, most browser extensions ask for access to 'read and change site data', allowing them to do almost anything with your data. Chrome extensions are always automatically updated, so a previously safe extension may update later to include malicious code. Most browser extensions are not trying to steal your data, but you should be aware that they can.

#### Stay safe by: {#browser-extention-safety}

- Only install browser extensions from trusted sources
- Removing unused browser extensions
- Install Chrome extensions locally to stop auto-updating (Advanced)

[More on the risks of browser extentions](https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/)

<Divider />

## Crypto Security 101 {#crypto-security}

> ### Upgrade your knowledge {#upgrade-your-knowledge}

Reviewer 3: "Level-up"? Small thought, but personally always pulled in more then this terminology is used
```suggestion
### Level up your knowledge {#level-up-your-knowledge}
```
Feel free to ignore =)

One of the biggest reasons people get scammed in crypto generally is a lack of understanding. For example, if you don't understand that the Ethereum network is decentralized and owned by no one, then it's easy to fall prey to someone pretending to be a customer service agent that promises to return your lost ETH in exchange for your private keys. Educating yourself on how Ethereum works is a worthwhile investment.

<DocLink to="/what-is-ethereum/" title="What is Ethereum?" />

<DocLink to="/eth/" title="What is ether?" />
<Divider />

## Wallet Security {#wallet-security}

> ### Don't give out your Private keys {#protect-private-keys}

Reviewer 4: ```suggestion
### Don't give out your private keys {#protect-private-keys}
```
I don't think capitalization is needed here?

**Never, for any reason, share your private keys!**

The private key to your wallet acts as a password to your Ethereum wallet. It is the only thing stopping someone who knows your wallet address from draining your account of all of its assets!

<DocLink to="/wallets/" title="What's an Ethereum wallet?" />

#### Don't take screenshots of your seed phrases/private keys {#screenshot-private-keys}

By screenshotting your seed phrases or private keys, you risk syncing them to the cloud and potentially making them accessible to hackers. Obtaining private keys from the cloud is a common attack vector for hackers.

> ### Use a Hardware Wallet {#use-hardware-wallet}

Reviewer 4: ```suggestion
### Use a hardware wallet {#use-hardware-wallet}
```
Same here - I believe our approach is sentence case for headers.

A hardware wallet provides offline storage for private keys. They are considered the most secure wallet option for storing your private keys.

Keeping private keys offline massively reduces the risk of being hacked, even if a hacker gets control of your computer.

> 

Reviewer 3: Should we mention mnemonic phrases here? Pretty much all hardware wallets (and most software wallets) utilize BIP-39 now, and this random list of words may be very confusing to those new to the space. 

People may not understand the differences/similarities between private keys and a mnemonic, and perhaps a quick note to remind people that these words should be the most guarded data you have in the crypto space. Also, guarding the mnemonic long-term involves more than theft protection, but also protection from natural disasters and damage such as fires/floods/crush/electrical shock (ie. put these words on steel).

#### Try a hardware wallet: {#try-hardware-wallet}

- [Ledger](https://www.ledger.com/)
- [Trezor](https://trezor.io/)

### Double check transactions before sending {#double-check-transactions}

> Accidently sending crypto to the wrong wallet address is a common mistake. **A transaction sent on Ethereum is irrevesible.** Unless you know the address owner and can convince them to send you your fund back, there will be no way for you to retrieve your funds.

Reviewer 3: ```suggestion
Accidentally sending crypto to the wrong wallet address is a common mistake. **A transaction sent on Ethereum is irreversible.** Unless you know the address owner and can convince them to send you your fund back, there will be no way for you to retrieve your funds.
```

### Set smart contract spend limits {#spend-limits}

When interacting with smart contracts, do not allow unlimited spend limits. An unlimited spend could enable the smart contract to drain your wallet. Instead, set spending limits to only the amount necessary for the transaction.

Many Ethereum wallets offer limits protection to safeguard against accounts being drained.

[Explore wallets with limits protection](/wallets/find-wallet/?filters=has_limits_protection)

<Divider />

## Common scams {#common-scams}

Todo:

No one is going to give you free or discounted ETH!

### Giveaway scam {#giveaway}

One of the most common scams in cryptocurrency is the giveaway scam. The giveaway scam can take many forms, but the general premise is that if you send ETH to the provided wallet address, you will receive your ETH back but doubled. *For this reason, it is also known as the 2-for-1 scam.*

These scams usually stipulate a limited time of opportunity to claim the giveaway to encourage poor decision-making and create a sense of false urgency.

#### Social media hacks {#social-media-hacks}

> A high-profile version of this occurred in July 2020, when the Twitter accounts of prominent celebrities and organisations got hacked. The hacker simultaneously posted a Bitcoin giveaway on the hacked accounts. Although the deceptive tweets were quickly noticed and deleted, the hackers still managed to get away with 11 bitcoin (or $500,000 as of September 2021)

Reviewer 3: ```suggestion
A high-profile version of this occurred in July 2020, when the Twitter accounts of prominent celebrities and organizations got hacked. The hacker simultaneously posted a Bitcoin giveaway on the hacked accounts. Although the deceptive tweets were quickly noticed and deleted, the hackers still managed to get away with 11 bitcoin (or $500,000 as of September 2021)
```
